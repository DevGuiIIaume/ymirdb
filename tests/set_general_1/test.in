set a 1 2 3
set b a 4 5 6
set c b 7 8 9
list entries
bye
