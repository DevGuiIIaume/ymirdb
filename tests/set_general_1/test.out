> ok

> ok

> ok

> c [b 7 8 9]
b [a 4 5 6]
a [1 2 3]

> bye
